db/migrate/20191102150647_create_references.rb

```diff
@@ -0,0 +1,14 @@
+class CreateReferences < ActiveRecord::Migration[6.0]
+  def change
+    create_table :references do |t|
+      t.string 'email_address', null: false
```

Comment:
I guess we will need a bit more info about the referee, at least a name (so that we can put _Dear Bob_ etc. in the reference request email). But that detail can be added later.

Reply:
Yes, I think the complete model can be driven out by the candidate form work. I wanted to just introduce enough to get going in this PR.